user-operator/src/test/java/io/strimzi/operator/user/model/KafkaUserModelTest.java

```diff
@@ -226,9 +226,9 @@ public void testGenerateSecretGeneratesCertificateWhenNoSecretExistsProvidedByUs
         Secret generated = model.generateSecret();
 
         assertThat(new String(model.decodeFromSecret(generated, "ca.crt")), is("clients-ca-crt"));
-        assertThat(new String(model.decodeFromSecret(generated, "user.crt")), is("crt file"));
-        assertThat(new String(model.decodeFromSecret(generated, "user.key")), is("key file"));
-        assertThat(new String(model.decodeFromSecret(generated, "user.p12")), is("key store"));
+        assertThat(new String(model.decodeFromSecret(generated, "user.crt")), is(MockCertManager.userCert()));
```

Comment:
> This nicely demonstrates the points I made on MockCertManager. You replaced something clearly mocked with something what is not valid anyway, but in a very sneaky way.

@scholzj for this comment , I'm hoping that `MockCertManager.userCert()` makes it clear this is a mock, given that having a real certificate is needed so I opted in for unifying how the tests get the expected values from the MockCertManager instead of hardcoding them sometimes and using getters in others.